### ISSUE TITLE
Cannot pass configs from commandline

### ISSUE THREAD

**Issue by claudiu-cristea** [openeuropa/task-runner#138]
Suppose this dynamic command in runner.yml:

```
some:
  config: 1
commands:
  some:command:
    - echo "Value is ${some.config}"
```

Theoretically, it should be possible to set a config on runtime, in this way:

```
run some:command -Dsome.config=2
```

Which you expect to echo: `Value is 2`

But it doesn't work!

**Comment by geek-merlin** [comment]
Yes i also noted this is documented somewhere (forgot where) but does not work.

Re-thinking it, i prefer the **explicit** way of having command args and options update defined places in config, so proposing WONTFIX.
